src/backend/debugging.md

```diff
@@ -222,9 +222,9 @@ really helpful for this.
 
 1. Once you have some LLVM IR for the problematic code (see above), you can
 create a minimal working example with Godbolt. Go to
-[gcc.godbolt.org](https://gcc.godbolt.org).
+[llvm.godbolt.org](https://llvm.godbolt.org).
 
-2. Choose `LLVM-IR` as programming language.
+2. Choose `LLVM-IR` as programming language if not already selected.
```

Comment:
I would just remove this line

Reply:
ok :)

Reply:
I meant the whole line, since `llvm.godbolt.org` makes it obsolete

Reply:
Oh. I initially wondered also. You'll get LLVM-IR by default, but if you change the language, close the tab, reopen it, the site will pick your last state and give you whatever you had when closing the tab, hence the initial change `if not already selected`.